myriad-scheduler/src/main/java/org/apache/myriad/webapp/MyriadWebServer.java

```diff
@@ -36,17 +38,19 @@
   private final Server jetty;
   private final Connector connector;
   private final GuiceFilter filter;
+  private final MyriadConfiguration myriadConf;
 
   /**
    * Status codes for MyriadWebServer
    */
   public enum Status {STARTED, RUNNING, STOPPED, FAILED, UNKNOWN}
   
   @Inject
-  public MyriadWebServer(Server jetty, Connector connector, GuiceFilter filter) {
+  public MyriadWebServer(Server jetty, Connector connector, GuiceFilter filter, MyriadConfiguration myriadConf) {
```

Comment:
If all we're using is isSecurityEnabled maybe just pass a boolean, if you think we'll need more of the config later leave it.